containers/unit-tests/setup.sh

```diff
@@ -56,7 +56,8 @@ apt-get update
 apt-get install -y --no-install-recommends eatmydata
 DEBIAN_FRONTEND=noninteractive eatmydata apt-get install -y --no-install-recommends ${dependencies}
 
-adduser --system --gecos "Builder" builder
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=679746
```

Comment:
Thanks for tracking that down.  The bug link probably belongs in the commit message rather than in a comment, though.